docs/make.bat

```diff
@@ -0,0 +1,242 @@
+@ECHO OFF

```

Comment:
Wondering if the make scripts should be consolidated to a single CMake project, in a future PR.

Reply:
Those are purely the output from Sphinx project generator, can be changed/remove at will/wish!